GameData/ROEngines/PartConfigs/RaptorVac_RE.cfg

```diff
@@ -73,12 +73,12 @@ PART
 	
 	MODULE
 	{
-	    name = ModuleGimbal
-	    gimbalTransformName = gimbal
+		name = ModuleGimbal
+		gimbalTransformName = gimbal
 		gimbalRange = 2.0
- 	    gimbalResponseSpeed = 10
- 	    useGimbalResponseSpeed = true
-    }
+		gimbalResponseSpeed = 10
+		useGimbalResponseSpeed = true
```

Comment:
Is it correct to set these gimbal parameters here instead of in the RO engine config?